nilearn/datasets/tests/test_atlas.py

```diff
@@ -229,8 +230,23 @@ def test_fetch_atlas_destrieux_2009():
 @with_setup(setup_mock, teardown_mock)
 @with_setup(tst.setup_tmpdata, tst.teardown_tmpdata)
 def test_fetch_atlas_msdl():
+    datadir = os.path.join(tst.tmpdir, 'msdl_atlas')
+    os.mkdir(datadir)
+    os.mkdir(os.path.join(datadir, 'MSDL_rois'))
+    data_dir = os.path.join(datadir, 'MSDL_rois', 'msdl_rois_labels.csv')
+    with open(data_dir, 'w') as csv_file:
+        fieldnames = ['x', 'y', 'z', 'name', 'net_name']
+        writer = csv.DictWriter(csv_file, fieldnames=fieldnames)
+        writer.writeheader()
```

Comment:
I am not sure whether we really care about Python 2.6 failure here. @AlexandreAbraham any ideas ?